habitat-baselines/habitat_baselines/common/tensorboard_utils.py

```diff
@@ -128,10 +128,30 @@ def __init__(
             raise ValueError(
                 "Requested to log with wandb, but wandb is not installed."
             )
+
         if resume_run_id is not None:
             wb_kwargs["id"] = resume_run_id
             wb_kwargs["resume"] = "must"
 
+        else:
+            wandb_id = wandb.util.generate_id()
```

Comment:
What are all these changes for? Wandb should support resuming on main now already.

Reply:
Yeah, these are for using the same wandb logs after resuming a previous run (after a reque or for evaluation). It looks like this is not supported on the main. The current main is creating separate wandb runs.

Reply:
In line with the recent changes in `main`, I made the fixes for loading the run_id from the resume state that gets saved.